pre_release.sh

```diff
@@ -71,7 +71,7 @@ if [ -n "${OTEL_TAG}" ]; then
     TMPDIR=$(mktemp -d "/tmp/otel-contrib.XXXXXX") || exit 1
     trap "rm -fr ${TMPDIR}" EXIT
     (cd "${TMPDIR}" && go mod init tagtest)
-    # requires go 1.14 for support of '-modfile'
+    # requires go 1.15 for support of '-modfile'
```

Comment:
Not sure that this version requirement changes, even if it's prior to the MSV.

Reply:
Sure

```suggestion
    # requires go 1.14 for support of '-modfile'
```